tutorials/time-series-modeling/index.md

```diff
@@ -0,0 +1,471 @@
+challenging. Note that in the previous example there was a well-formed
+data point for every single month—nothing was missing or obviously
+erroneous. It is almost inevitable that you will not get this same level of
+precision from remote sensing data.
+[Landsat-8](https://developers.google.com/earth-engine/datasets/catalog/LANDSAT_LC08_C02_T1_L2)
+has 16-day temporal resolution, but depending on the area,
+removing cloudy pixels will remove a significant portion. In a test area in
+the Galapagos Islands, over 85% of the data was removed due to cloud masking
+or atmospheric conditions. Issues such as the
+[Landsat 7 Scan Line Corrector malfunction](https://www.usgs.gov/faqs/what-landsat-7-etm-slc-data)
+might prevent a cohesive time series dataset depending on your time period of
+research. Also, while the time series example we used involved measured
+values on the same scale throughout the time series (e.g., a gigawatt is the
+same unit of measurement throughout the entire time series), with remote
+sensing we often run into situations where the magnitude of measurement
+changes. If we are researching winter crop yield and an image is collected
+right after a heavy snowfall, how do we compare this value? Do we keep this
+data or remove it? Additionally, atmospheric conditions can skew the visual
+results, where the hue of the vegetation changes drastically from image to
+image due to atmospheric conditions (fog, ground moisture, cloud cover).
+
+It is important to understand the characteristics of both your data and what
+you are trying to measure. Building a time series model to understand
+cyclical changes in vegetation can provide useful information in
+understanding crop yield, but if you do not account for issues in the data,
+you can end up building a time series model with erroneous results. Many time
+series modeling tools, such as
+[ARIMA modeling](https://otexts.com/fpp2/arima.html), are not directly
+applicable in certain settings due to missing data, non-standard collection
+periods and varying intensity due to atmospheric conditions. In this tutorial,
+we will focus on understanding linear trends and harmonic modeling.
+
+### Multi-temporal data in Earth Engine
+
+Time series data in Earth Engine are represented as a series of images called
+'[Image Collections](https://developers.google.com/earth-engine/guides/ic_creating)'.
+As a result of the complicating factors in remote sensing discussed earlier,
+analyzing time series in Earth Engine is unlike time series modeling in
+traditional methods. From a programming sense, we will join data together to
+define temporal relationships between collection items and build functions to
+reduce this time.
+
+First, some very basic mathematical notation for time series. A time series
+is an array of the value being measured, sorted chronologically: {
+$\textbf{p}_{t}  = t_{0} + t_{1}... t_{N}$ }, where each *t* is the given
```

Comment:
Are the braces (`{` `}`) intended to be formatted as part of the equation? Maybe include them inside `$...$`.

Reply:
Not necessary so I just removed them.